frontend/src/components/Schedule/Schedule.tsx

```diff
@@ -142,6 +136,7 @@ const Schedule = () => {
           </div>
         </div>
       </div>
+      <button className={styles.more} onClick={() => setViewMore(!viewMore)}>view more</button>
```

Comment:
I think the button text should change to "view less" after the table expands

Reply:
Good catch.